### PR TITLE
Add generic jenkins job for email-notification application

### PR DESCRIPTION
**Pull request by dmkov**
Hi Matt,

This is a new jenkins job for email-notifier. Build.sh was added there but was not merged into dockerization branch yet.

### CODE REVIEW

File: jobs/app-jobs.yml [hunk 1]
```diff
@@ -88,6 +88,13 @@
     jobs:
       - 'generic-template'
 
+- project:
+    name: email-notifier
```
Comment (mattdrees) [inline jobs/app-jobs.yml]: this should match the github repo name

Comment (mattdrees) [reply jobs/app-jobs.yml]: (which is confusing, because it doesn't match the maven artifact, or what I usually say when I talk about the project; sorry)

Comment (dmkov) [reply jobs/app-jobs.yml]: Done! Updated it here and in ecr_config repository. Thank you for letting me know.

**Comment by cru-Luis-Rodriguez** [comment]
replaced by https://github.com/CruGlobal/jenkins-jobs/pull/114

**Pull request closed without merging**